app/views/settings_dialog.py

```diff
@@ -686,6 +688,51 @@ def _do_todds_tab(self) -> None:
         )
         group_layout.addWidget(self.todds_overwrite_checkbox)
 
+    def _do_themes_tab(self) -> None:
+        tab = QWidget()
+        self.tab_widget.addTab(tab, "Theme")
+
+        tab_layout = QVBoxLayout(tab)
+        tab_layout.setAlignment(Qt.AlignmentFlag.AlignTop)
+
+        theme_group_box = QGroupBox("Themes")  # Added a title for the group box
+        tab_layout.addWidget(theme_group_box)
+
+        theme_group_layout = QVBoxLayout()
+        theme_group_box.setLayout(theme_group_layout)
+        theme_group_box.setFont(GUIInfo().emphasis_font)
+
+        self.enable_themes_checkbox = QCheckBox(
+            "Enable to use theme / stylesheet instead of system Theme"
+        )
+        self.enable_themes_checkbox.setToolTip(
+            "To add your own theme / stylesheet \n\n"
+            "1) Create a new-folder in 'themes' folder in your 'RimSort' config folder \n"
+            "2) Using the default 'RimPy' theme copy it to the folder you created \n"
+            "3) Edit the copied 'style.qss' as per your imagination \n"
+            "4) Start 'RimSort' and select your theme from dropdown \n"
+            "5) Restart 'RimSort' to apply the theme \n\n"
+            "NOTE \n"
+            "Name of folder will be used as name of the theme and any invalid theme will be ignored \n"
+        )
+        theme_group_layout.addWidget(self.enable_themes_checkbox)
+
+        self.themes_combobox = QComboBox()
+        self.themes_combobox.setSizePolicy(
+            QSizePolicy.Policy.Maximum, QSizePolicy.Policy.Preferred
+        )
+
+        theme_controller = ThemeController()
+        available_themes = list(
+            theme_controller.get_supported_themes()
+        )  # Ensure it's a list
+        self.themes_combobox.addItems(available_themes)
+        theme_group_layout.addWidget(self.themes_combobox)
+
+        self.theme_location_open_button = QToolButton()
+        self.theme_location_open_button.setText("Open Theme Location")
+        theme_group_layout.addWidget(self.theme_location_open_button)
```

Comment:
Move this to the same line as the drop down menu. Also, it should really be opening the user themes folder, not the theme currently being used. The idea is that you click it to go to the directory to install new themes or remove them.

Reply:
ok done